agent_api_rest/src/issuance/credentials.rs

```diff
@@ -149,6 +197,38 @@ pub(crate) async fn all_credentials(State(state): State<IssuanceState>) -> Resul
     Ok((StatusCode::OK, Json(all_credentials)).into_response())
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct PatchCredentialEndpointRequest {
+    pub credential_status: StatusType,
+}
```

Comment:
We are not consistently doing this right now (at all), but we need to make sure that for the UniCore-specific API everything is camelCase (through `#[serde(rename_all = "camelCase")]`) (see: https://github.com/impierce/ssi-agent/pull/199#discussion_r2239377454)